code/__HELPERS/logging/_logging.dm

```diff
@@ -10,6 +10,31 @@
 	msg = "## NOTICE: [msg]"
 	log_world(msg)
 
+#define SET_SERIALIZATION_SEMVER(semver_list, semver) semver_list[type] = semver
+#define CHECK_SERIALIZATION_SEMVER(wanted, actual) (__check_serialization_semver(wanted, actual))
```

Comment:
These need doc comments

Reply:
You didn't document these two defines still though.